app/Policies/CloudInitScriptPolicy.php

```
@@ -0,0 +1,65 @@
+    /**
+     * Determine whether the user can delete the model.
+     */
+    public function delete(User $user, CloudInitScript $cloudInitScript): bool
+    {
+        return $user->isAdmin();
+    }
+
+    /**
+     * Determine whether the user can restore the model.
+     */
+    public function restore(User $user, CloudInitScript $cloudInitScript): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$cloudInitScript'. (undefined)

(UnusedFormalParameter)